extensions/wikia/Recirculation/templates/Recirculation_discussions.php

```diff
@@ -0,0 +1,33 @@
+<section class="mcf-card mcf-card-discussions">
+	<header class="mcf-card-discussions__header">
+		<span>Latest Discussions</span>
+		<a href="<?= $discussionsUrl ?>" class="mcf-card-discussions__link">View All
+			<?= DesignSystemHelper::renderSvg(
+				'wds-icons-arrow',
+				'wds-icon wds-icon-small mcf-card-discussions__link-icon'
+			) ?></a>
+	</header>
+	<ul>
+		<?php foreach ( $posts as $post ): ?>
+			<li class="mcf-card-discussions__item">
+				<a href="/User:<?= $post->author ?>" class="mcf-card-discussions__user-info">
```

Comment:
Use https://github.com/Wikia/app/blob/3013e9eec7bb1abe9a19f020a439445f341c5abe/includes/wikia/services/AvatarService.class.php#L64-L64 to get profile url